.travis.yml

```diff
@@ -5,8 +5,18 @@ services:
 
 language: java
 
-jdk:
-  - oraclejdk8
+matrix:
+  include:
+    - jdk: openjdk8
+    - jdk: openjdk9
+      before_install:
+        - rm "${JAVA_HOME}/lib/security/cacerts"
+        - ln -s /etc/ssl/certs/java/cacerts "${JAVA_HOME}/lib/security/cacerts"
+    - jdk: openjdk10
+      before_install:
+        - rm "${JAVA_HOME}/lib/security/cacerts"
+        - ln -s /etc/ssl/certs/java/cacerts "${JAVA_HOME}/lib/security/cacerts"
```

Comment:
whats this about?

Reply:
Ah, it's in my second commit's message.

Reply:
I only needed this to be able to use SNAPSHOT deps. I'll remove once docker-client is released.

Reply:
ah sorry I missed that commit message. It must be Monday

Reply:
Oops, I was also mistaken. These certs need to be mounted regardless of whether you use SNAPSHOT deps or not from sonatype. It's bc openjdk9 and 10 lack these certs...